app/lib/cfe_connection.rb

```diff
@@ -55,6 +55,13 @@ def create_employment(assessment_id, employment_income)
     create_record(assessment_id, "employments", employment_income:)
   end
 
+  CFE_OUTGOING_FREQUENCIES = {
```

Comment:
Not necessarily a blocker but a note that usually (`OUTGOINGS_ATTRIBUTES` now, the form name, the handler etc) we  used `outgoings` but sometimes (`OUTGOING_ATTRIBUTES` previously, and here) we use 'outgoing' singular. If this isn't deliberate, we should be consistent with 'outgoings'.

However, if it _is_ to distinguish between 'outgoings' as in money paid out and 'outgoing' as in regarding an external call to CFE... 😅